packages/sdk/contractkit/README.md

```diff
@@ -63,29 +69,28 @@ celo-blockchain has two initial coins: CELO and cUSD (stableToken).
 Both implement the ERC20 standard, and to interact with them is as simple as:
 
 ```ts
-const goldtoken = await kit.contracts.getGoldToken()
+// get the CELO contract
+const celoToken = await kit.contracts.getGoldToken()
 
-const balance = await goldtoken.balanceOf(someAddress)
+// get the cUSD contract
+const stableToken = await kit.contracts.getStableToken()
+
+const celoBalance = await goldToken.balanceOf(someAddress)
+const cusdBalance = await stableToken.balanceOf(someAddress)
 ```
 
 To send funds:
 
 ```ts
 const oneGold = kit.web3.utils.toWei('1', 'ether')
```

Comment:
this line also still works, but similar to my comment above, would change to `kit.connection.web3.utils.toWei`